fast_plotter/utils.py

```diff
@@ -78,8 +78,11 @@ def split_df(df, first_values, level=0):
     if not first_values:
         return None, df
     second = df.drop(first_values, level=level)
-    second_values = second.index.unique(level=level)
-    first = df.drop(second_values, level=level)
+    if not second.empty:
```

Comment:
This change is just so that single datasets (i.e. just signal or just data) can be plotted. I think you mentioned a fix you implemented but didn't merge in, Ben; that'd presumably do the same job in a 'nicer' way.